tasks/internal/filter-already-released-advisory-images-task/filter-already-released-advisory-images-task.yaml

```diff
@@ -122,13 +122,14 @@ spec:
           exit 0
         fi
 
+        EXISTING_CONTENT=/tmp/existing_content.json
         for ADVISORY_SUBDIR in $EXISTING_ADVISORIES; do
           ADVISORY_FILE="${ADVISORY_BASE_DIR}/${ADVISORY_SUBDIR}/advisory.yaml"
-          EXISTING_CONTENT=$(yq -o=json '.spec.content.images // []' "$ADVISORY_FILE")
+          yq -o=json '.spec.content.images // []' "$ADVISORY_FILE" > "$EXISTING_CONTENT"
 
           echo "Comparing against: $ADVISORY_FILE"
 
-          CONTENT_IMAGES=$(echo "$CONTENT_IMAGES" | jq --argjson existing "$EXISTING_CONTENT" '
+          CONTENT_IMAGES=$(echo "$CONTENT_IMAGES" | jq --argfile existing "$EXISTING_CONTENT" '
```

Comment:
From the jq manual I see that argfile has been deprecated since 1.5, it advises to use [--slurpfile](https://jqlang.org/manual/#:~:text=%2D%2Dslurpfile%20variable%2Dname%20filename%3A) instead.

Reply:
Good catch. It looks like it is removed completely in 1.8, since it's not even listed there? I checked and we are currently on version 1.6. So yeah, we should use --slurpfile. Fixed.

Reply:
It looks like there are some differences between the two unfortunately - slurpfile creates a json array, so it should be `$existing[0]` I guess?

Reply:
Ah, right, thanks. I see Scott already used it like that in the collectors. Should be fixed now.